app/assets/images/index.js

```diff
@@ -0,0 +1,10 @@
+export const images = {
```

Comment:
What about creating a dedicated reusable/presentational component for SVG "ImageSvg" and then an extra layer on top of it for each pic ? "ImageIconCheck" etc.

While you'll gonna use pics and anyway import these images why don't you just import directly the ready-made component?

Reply:
I think I understand what you're suggesting, but a pseudo-code example would be helpful.

Reply:
`export function Icon({ ...rest }) {
  return (
      <svg> </svg>
  )
}`

Reply:
Got it.  Since I'll be removing those svg icons & instead use the react-native-svg-icon lib that's already installed, I'm going to hold off on implementing this suggestion.  

Although, requiring each image as a property in the `images` object, would cause ALL of the required images to be stored in memory whenever `images` gets imported into a component - so I think this could still be refactored for performance reasons.  I'll add this to a separate issue for later.